buckw

```diff
@@ -213,10 +213,15 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
-        SKIP_OKBUCK=true
-    fi
+    for arg in "$@"
```

Comment:
The `comm` command seems to take only files as input. The example uses process substitution to read a process's output as a file. I'm not seeing a way to do it with `comm`?